### PR TITLE
Docs: update customQuery types in docs

### PR DESCRIPTION
**Pull request by lukaszjedrasik**
### Related Issues

[#EN-160](https://vsf.atlassian.net/browse/EN-160?atlOrigin=eyJpIjoiYzYxZGUzZDUxNzQzNDI2MWI0YWRiNjNkN2MwYjY3OTgiLCJwIjoiaiJ9)

### Short Description of the PR

Update `customQuery` types to show how they should look like.

### Pull Request Checklist

- [ ] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.
- [ ] I have tested my Pull Request on production build and (to my knowledge) it works without any issues

- [ ] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

### CODE REVIEW

**Review by filrak — APPROVED** [review]

File: packages/core/docs/commercetools/composables/use-billing.md [hunk 1]
```diff
@@ -47,7 +47,9 @@ type Address = {
   externalId?: Maybe<Scalars["String"]>;
   key?: Maybe<Scalars["String"]>;
 };
-type CustomQuery = Record<string, string>
+type CustomQuery = {
+  updateCart: string
+}
```
Comment (filipsobol) [inline packages/core/docs/commercetools/composables/use-billing.md:52]: I don't think it's the case for both `load` and `save`

File: packages/core/docs/commercetools/composables/use-cart.md [hunk 2]
```diff
@@ -87,7 +87,9 @@ type ProductVariant = {
   attributeList: Array<Attribute>;
 }
 
-type CustomQuery = Record<string, string>
+type CustomQuery = {
+  updateCart: string
+}
```
Comment (filipsobol) [inline packages/core/docs/commercetools/composables/use-cart.md:92]: I don't think it's the case for both `load` and `save`. Also `load` still has `customQuery?: any`

File: packages/core/docs/commercetools/composables/use-shipping-provider.md [hunk 3]
```diff
@@ -38,7 +38,9 @@ type ShippingMethod = Versioned & {
   taxCategory?: Maybe<TaxCategory>;
 };
 
-type CustomQuery = Record<string, string>
+type CustomQuery = {
+  updateCart: string
+}
```
Comment (filipsobol) [inline packages/core/docs/commercetools/composables/use-shipping-provider.md:43]: I don't think it's the case for both `load` and `save`.

File: packages/core/docs/commercetools/composables/use-shipping.md [hunk 4]
```diff
@@ -48,7 +48,9 @@ type Address = {
   key?: Maybe<Scalars["String"]>;
 };
 
-type CustomQuery = Record<string, string>
+type CustomQuery = {
+  updateCart: string
+}
```
Comment (filipsobol) [inline packages/core/docs/commercetools/composables/use-shipping.md:53]: I don't think it's the case for both `load` and `save`.

File: packages/core/docs/commercetools/composables/use-user-shipping.md [hunk 5]
```diff
@@ -79,7 +79,7 @@ interface ShippingAddressUpdateParams {
 
     - `address: ShippingAddressSetDefaultParams`
 
-    - `customQuery?
+    - `customQuery?`
```
Comment (filipsobol) [inline packages/core/docs/commercetools/composables/use-user-shipping.md:82]: Missing type (same for `addAddress`, `deleteAddress` and `updateAddress`).

**Review by filipsobol — CHANGES_REQUESTED** [review]

File: packages/core/docs/commercetools/composables/use-cart.md (hunk 2 above)
Comment (lukaszjedrasik) [reply packages/core/docs/commercetools/composables/use-cart.md:98]: Yup, `load` still has `customQuery?: any` but inside documentation I changed it to `customQuery?: customQuery`

**Review by andrzejewsky — APPROVED** [review]

File: packages/core/docs/commercetools/composables/use-billing.md [hunk 6]
```diff
@@ -12,6 +12,12 @@
 - `load` - function for fetching billing address. When invoked, it requests data from the API and populates `billing` property. This method accepts a single optional `params` object. The `params` has the following option:
 
     - `customQuery?: CustomQuery`
+  
+```ts
+type CustomQuery = {
+  getBasicProfile: string
```
Comment (filipsobol) [inline packages/core/docs/commercetools/composables/use-billing.md:18]: `getBasicProfile` for guests and `getFullProfile` for logged-in users

File: packages/core/docs/commercetools/composables/use-cart.md [hunk 7]
```diff
@@ -61,7 +61,13 @@ type Cart = {
 
 - `load` - function required to fetch cart from a server or create brand new if it doesn't exist. This method accepts a single `params` object. The `params` has the following option:
 
-    - `customQuery?: any`
+    - `customQuery?: customQuery`
+  
+```ts
+type CustomQuery = {
+  getBasicProfile: string
```
Comment (filipsobol) [inline packages/core/docs/commercetools/composables/use-cart.md:68]: `getBasicProfile` for guests and `getFullProfile` for logged-in users

File: packages/core/docs/commercetools/composables/use-shipping-provider.md [hunk 8]
```diff
@@ -12,6 +12,12 @@
 - `load` - function for fetching shipping method. When invoked, it requests data from the API and populates the `response` key inside the `state` property. This method accepts a single optional `params` object. The `params` has the following option:
 
     - `customQuery?: CustomQuery`
+  
+```ts
+type CustomQuery = {
+  getBasicProfile: string
```
Comment (filipsobol) [inline packages/core/docs/commercetools/composables/use-shipping-provider.md:18]: `getBasicProfile` for guests and `getFullProfile` for logged-in users

File: packages/core/docs/commercetools/composables/use-shipping.md [hunk 9]
```diff
@@ -12,6 +12,12 @@
 - `load` - function for fetching shipping address. When invoked, it requests data from the API and populates `shipping` property. This method accepts a single optional `params` object. The `params` has the following option:
 
     - `customQuery?: CustomQuery`
+  
+```ts
+type CustomQuery = {
+  getBasicProfile: string
```
Comment (filipsobol) [inline packages/core/docs/commercetools/composables/use-shipping.md:18]: `getBasicProfile` for guests and `getFullProfile` for logged-in users

**Review by filipsobol — CHANGES_REQUESTED** [review]

**Review by filipsobol — APPROVED** [review]

**Pull request merged**